sdk/src/main/java/com/horizen/account/api/rpc/request/RpcId.java

```diff
@@ -7,6 +7,8 @@ public class RpcId {
     private Long longId;
     private String stringId;
 
+    public RpcId() {}
```

Comment:
do we really need this constructor?

Reply:
Yes, we need this default constructor because we need to initialize a `RpcId` object to correctly serialize a null response id